build.rs

```diff
@@ -123,6 +123,7 @@ fn main() {
         println!("cargo:rerun-if-changed=proto/google/pubsub/v1/pubsub.proto");
         println!("cargo:rerun-if-changed=proto/google/rpc/status.proto");
         println!("cargo:rerun-if-changed=proto/vector.proto");
+        println!("cargo:rerun-if-changed=proto/vector_event_log.proto");
```

Comment:
do we need to add this file in code base? 
how do we plan to use it? 
can we achieve same thing by using proto file in vrl? https://vector.dev/docs/reference/vrl/functions/#encode_proto

Reply:
+1

I think updating/understanding the whole vector code base will likely be suboptimal at this stage since we haven't done a lot of feature work inside vector. Might be simpler to just capture the file emission events for now since those are the most critical (best bang for buck)